__tests__/app.test.js

```diff
@@ -32,6 +32,30 @@ describe('/api/categories', () => {
                 })
             })
         })
+    })
+})
+
+describe('/api/users', () => {
+    describe('GET requests', () => {
+        describe('GET /api/users', () => {
+            test('Happy path', () => {
+                return request(app).get('/api/users').expect(200).then(({body}) => {
+                    expect(body.length).toBe(4) 
+                    body.forEach(user => {
+                        expect(user).toEqual(expect.objectContaining({
+                            username: expect.any(String),
+                            name: expect.any(String),
+                            avatar_url: expect.any(String)
+                        }))
+                    })
+                })
+            })
+        })
+    })
+})
```

Comment:
This is for a different endpoint and should be not in this PR. I will allow it this time but next time make sure you are separating your functionality/tests by the branches they are meant for